rasa/nlu/featurizers/featurzier.py

```diff
@@ -10,6 +10,21 @@
 )
 
 
+def sequence_to_sentence_embedding(
+    features: Union[List[List[float]], scipy.sparse.spmatrix], method: Text = "mean"
```

Comment:
why don't we keep dense features as `np.ndarray`?

Reply:
we do, will change the type